dockerfiles/hack/run-integration.py

```diff
@@ -50,13 +50,20 @@
 logging.basicConfig(level=logging.INFO,
                     handlers=HANDLERS)
 
+def _parse_dry_run_flag(dry_run: str) -> Optional[str]:
+    dry_run_options = ['--dry-run', '--no-dry-run']
+    if dry_run is not None and dry_run not in dry_run_options:
+        logging.error('Invalid DRY_RUN option given: "%s". Only the following options are allowed: %s', dry_run, dry_run_options)
+        sys.exit(1)
+    if dry_run in ['--dry-run', '--no-dry-run']:
```

Comment:
DRY:
```suggestion
    if dry_run in dry_run_options:
```
(or better, `if dry_run:` since you've checked everything else in the block above)

Reply:
Good catch!
Fixed